low-level/files/opt/seagate/sspl/conf/kafka.service

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Needs to be under low-level/files/opt/seagate/sspl/setup/standalone/

Reply:
Done